src/plugins/maps_dashboards/common/index.ts

```diff
@@ -3,5 +3,15 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-export const PLUGIN_ID = 'mapsDashboards';
+export const PLUGIN_ID = 'maps-dashboards';
 export const PLUGIN_NAME = 'Maps';
+
+// This style URL is only used for development, will replace with production vector tile service url.
+export const MAP_VECTOR_TILE_URL =
+  'https://dldbnqfps17cd.cloudfront.net/styles/basic-preview/compressedstyle.json';
+
+export const MAP_INITIAL_STATE = {
+  lng: 11,
+  lat: 49,
+  zoom: 2,
+};
```

Comment:
please specify what these coordinates represents.

Reply:
Will update to 
```
  lng: 0,
  lat: 0,
  zoom: 1,
```